docs/client-development/recipes.rst

```diff
@@ -188,9 +188,10 @@ The solution is to use some form of threading/concurrency offered by your langua
             from zgw_consumers.constants import APITypes
             from zgw_consumers.models import Service
             from zgw_consumers.concurrent import parallel
+            from zgw_consumers.client import build_client
 
-            zrc_client = Service.objects.filter(api_type=APITypes.zrc).get().build_client()
-            drc_client = Service.objects.filter(api_type=APITypes.drc).get().build_client()
+            zrc_client = build_client(Service.objects.get(api_type=APITypes.zrc))
+            drc_client = build_client(Service.objects.get(api_type=APITypes.drc))
 
             zaak_url = "https://test.openzaak.nl/zaken/api/v1/zaken/b604ea56-f01c-432e-8d61-fd4ab02893dc"
             zios: List[dict] = zrc_client.list("zaakinformatieobject", {"zaak": zaak_url})
```

Comment:
I think the usage of `.list` is also outdated, now it's using ape_pie which uses the same methods as `requests` (.get("zaakinformatieobjecten") in this case)